concrete/config/install/base/single_pages/dashboard.xml

```diff
@@ -875,13 +875,6 @@
                 </attributekey>
             </attributes>
         </page>
-        <page name="Security Options" path="/dashboard/system/permissions/security" filename="/dashboard/system/permissions/security.php" pagetype="" description="" package="">
```

Comment:
This change is not in your PR description, but it seems quite relevant to me. How will this effect installations that have for example overridden this page? Or added a page underneath?

Reply:
We remove the page in the migration: https://github.com/concrete5/concrete5/pull/7227/files#diff-d46c1010bb40b0eec37790f1ab5e0c3bR20 so an end use would likely need to reimplement their security page.

This seems like an important oversight if we were strictly semver, but I'd argue this is super low impact and not worth keeping a blank security page around if we don't need to

Reply:
And what happens if they've added a page underneath, for example? For add-ons it's quite common to install single pages underneath core system pages.